spec/workers/commit_monitor_handlers/commit_range/rubocop_checker/data/with_results_with_offenses/ruby_syntax_error.rb

```diff
@@ -1,3 +1,6 @@
 class RubySyntaxError
+  def initialize
+    @x = 1
+  end
```

Comment:
This technically wasn't needed, but since I did it above to avoid the EmptyClass, I did it here too.